axelrod/tests/strategies/test_darwin.py

```diff
@@ -15,25 +15,41 @@ class TestDarwin(TestPlayer):
         'stochastic': False,
         'makes_use_of': set(),
         'long_run_time': False,
-        'inspects_source': False,
+        'inspects_source': True,
         'manipulates_source': False,
         'manipulates_state': True
     }
 
+    def setUp(self):
```

Comment:
unfortunately, no.  this is overriding the method of unittest.TestCase.setUp which gets called before every test.